.github/workflows/nodejs.yml

```diff
@@ -5,18 +5,27 @@ on: [push]
 jobs:
   lint:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version: [12.x]
     steps:
-      - uses: actions/checkout@v1
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v1
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
         with:
-          node-version: ${{ matrix.node-version }}
-      - name: npm install and lint
-        run: |
-          npm ci
-          npm run lint --silent
+          node-version: '10'
```

Comment:
@eugene-manuilov shouldn't we use node 12 here?

Reply:
@ivankristianto node 10 is the current LTS version + some packages rely on it, so it's okay to stick with it for now. We can update it later.

Reply:
I'd rather go ahead and be more "future-proof" and just use node 12 as it is actually the current and active LTS. Node 10 is in maintenance mode.

https://nodejs.org/en/about/releases/

Are there any packages here that do not work with node 12?

Reply:
OK, let me explain it from another angle: i don't mind to change node version to 12, but i think we need to test the oldest maintainable version which is 10 as of now because there might be users who are still on it.